src/Handle.ts

```diff
@@ -45,6 +52,12 @@ export class Handle<T> {
     }
   }
 
+  pushMessage = (contents: any) => {
```

Comment:
We really need a way to specify private/internal methods - or at least have a better naming scheme. `message()` vs `pushMessage`() is pretty ambiguous.

Reply:
I agree with this but I didn't do it. :D